java/runtime/src/main/java/org/ray/runtime/object/ObjectStoreProxy.java

```diff
@@ -0,0 +1,172 @@
+package org.ray.runtime.object;
+
+import com.google.common.base.Preconditions;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import org.ray.api.exception.RayActorException;
+import org.ray.api.exception.RayException;
+import org.ray.api.exception.RayTaskException;
+import org.ray.api.exception.RayWorkerException;
+import org.ray.api.exception.UnreconstructableException;
+import org.ray.api.id.ObjectId;
+import org.ray.runtime.actor.NativeRayActor;
+import org.ray.runtime.context.WorkerContext;
+import org.ray.runtime.generated.Gcs.ErrorType;
+import org.ray.runtime.util.Serializer;
+
+/**
+ * A class that is used to put/get objects to/from the object store.
+ */
+public class ObjectStoreProxy {
```

Comment:
I think we merge this class to `ObjectStore` and make `ObjectStore` an abstract class.

Reply:
Yes, we can do that.